src/Numerics/Mesh/Elements.jl

```diff
@@ -115,4 +115,21 @@ function interpolationmatrix(
     I
 end
 
+"""
+    jacobip(α, β, np, x)
+
+Returns a `(nx, np+1)` array containing the `np+1` Jacobi polynomials of order `np`, 
+with parameter `(α, β)`, evaluated on 1D grid `x`.
+"""
+function jacobip(
+    α::FT,
+    β::FT,
+    np::Int,
+    x::AbstractArray{FT, 1},
+) where {FT <: AbstractFloat}
+    a, b = GaussQuadrature.jacobi_coefs(np, α, β)
+    V = GaussQuadrature.orthonormal_poly(x, a, b)
+    return V
+end
```

Comment:
```suggestion
"""
    jacobip(α, β, N, x)

Returns a `(nx, np+1)` array containing the `N+1` Jacobi polynomials of order `N`, 
with parameter `(α, β)`, evaluated on 1D grid `x`.
"""
function jacobip( α, β, N, x)
    a, b = GaussQuadrature.jacobi_coefs(N, α, β)
    V = GaussQuadrature.orthonormal_poly(x, a, b)
    return V
end
```
elsewhere we use `N` for polynomial order. Type information is not needed

Reply:
revised.